--- origsrc/webkitmm/webkit/webkitmm/private/Makefile.am	2009-06-09 14:55:31.000000000 -0500
+++ src/webkitmm/webkit/webkitmm/private/Makefile.am	2009-06-09 15:18:11.966257900 -0500
@@ -1,10 +1,10 @@
 ## Copyright (c) 2001
-## The libgdamm development team.
+## The webkitmm development team.
 
 include $(srcdir)/../../src/Makefile_list_of_hg.am_fragment
 files_built_h		= $(files_hg:.hg=_p.h)
 
-private_includedir	= $(includedir)/libgdamm-3.0/libgdamm/private
+private_includedir	= $(includedir)/webkitmm-1.0/webkitmm/private
 private_include_HEADERS	= $(files_built_h)
 
 maintainer-clean-local:
